apiary.apib

```diff
@@ -64,29 +90,45 @@ Destroying an account, and changing account properties
 like username and password, require to pass the user's
 password as `x-password` header.
 
-### Sign Up [PUT]
+### Sign Up [POST]
```

Comment:
I closed this PR as I made it too soon, there's another one over here #2 

Take a look through the [JSON-API spec](jsonapi.org/format/) first, it will probably answer a bunch of questions up front :)